WhatsNew/WhatsNew/Presentation/WhatsNewView.swift

```diff
@@ -124,6 +130,7 @@ public struct WhatsNewView: View {
                         .allowsHitTesting(false)
                         .padding(.top, isHorizontal ? 0 : 170)
                         .padding(.bottom, 8)
+                        .accessibilityIdentifier("whatsnew_pagecontrol")
```

Comment:
Suggestion to replace with `whats_new_page_control`

Reply:
Here `pagecontrol` is single entity.